packages/code-connect/icons/IconWrapper.figma.tsx

```diff
@@ -0,0 +1,31 @@
+import figma from '@figma/code-connect';
```

Comment:
Not this line but the file name - missed earlier that the file can be renamed from "IconWrapper" to just "Icon" to match [the component name](https://www.patternfly.org/components/icon).